crates/symbolicator-service/src/caching/config.rs

```diff
@@ -26,7 +26,7 @@ impl AsRef<str> for CacheName {
             Self::Cficaches => "cficaches",
             Self::PpdbCaches => "ppdb_caches",
             Self::ArtifactCaches => "artifact_caches",
-            Self::SourceMapCaches => "sourcesmap_caches",
+            Self::SourceMapCaches => "sourcemap_caches",
```

Comment:
kinda glad we didn’t actually start using the functionality with this typo in it :-D

Reply:
Noone would ever notice, which means that it was never there 🤫